docs/components/modeler/web-modeler/deploy-process-application.md

```diff
@@ -0,0 +1,66 @@
+---
+id: deploy-process-application
+title: Deploy and run your process application
+description: You can deploy your process application to a Zeebe cluster, or run your process application to test and debug it, and observe how it performs in a live environment..
+---
+
+import DeployImg from './img/process-applications/define-stages-deploy.png'
+import RunProcessApplicationImg from './img/process-applications/run-process-application.png'
+import DeployErrorImg from './img/process-applications/deploy-error.png'
+
+You can deploy your process application to a Zeebe cluster, and run your process application to test and debug it, and observe how it performs in a live environment.
+
+## Deploy your process application
+
+To deploy your process application:
+
+1. Open the [main process](create-a-process-application.md#main-process).
+1. Select **Deploy** to open the **Deploy process application** modal.
+   <p><img src={DeployImg} alt="Deploy a process application" /></p>
+1. Turn on the toggle for the cluster stage you want to deploy to.
+1. Perform any other actions as required, such as:
+   - Unpausing the chosen cluster if it has been auto-paused. Select **Resume**.
+   - Managing the cluster. Select **Manage**.
+   - [Defining the stages](process-application-pipeline.md#deployment-stages) of your development pipeline. Select **Define stages**.
+1. Select **Deploy** to deploy the process application to the selected cluster.
+
+All BPMN, DMN, and form files contained in the process application folder are deployed as a single bundle.
+
+:::note
+If any resource fails to deploy, the whole deployment [fails](#deployment-errors) and the cluster state remains unchanged. This safely ensures that a process application cannot be deployed incompletely or in an inconsistent state.
+:::
+
+## Run a process application
```

Comment:
Heads-up: In 8.6, I will start pushing users to use Play for this purpose rather than Run

Reply:
Should I remove the "Run" section in this page then? If so, do we need to replace it with a "Play" section, or just point people towards the Play feature docs I would guess?

Reply:
You don't need to make changes for now, but we'll need to revisit this for the 8.6.0 release

Between now and 8.6, it makes sense to deploy your process app since Play doesn't connect to a cluster
Starting with 8.6, we should tell people to use Play to run their process in a development cluster and only use the Deploy/Run modals for interacting with other stages

Reply:
Thanks for the heads up: I've created https://github.com/camunda/camunda-docs/issues/4182 for this work so it doesn't get missed.